### PR TITLE
Added BbCoreJS dependency

### PR DESCRIPTION
**Pull request by mickaelandrieu**
This is a suggest to use composer to install BbCoreJs front client into ToolbarBundle.

* Add the repository as a dependency
* Using Script Handler, move the dependency to the good destination

This wont fix all the "external" dependecy (bower, npm, etc), but this is a good start.

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -13,13 +13,36 @@
...
+                    "url": "https://github.com/backbee/BbCoreJs/archive/master.zip",
+                    "type": "zip"
+                }
+            }
+        }
+    ],
     "require": {
-        "backbee/backbee": "dev-master"
+        "php": ">=5.4.0",
+        "incenteev/composer-parameter-handler": "~2.1",
```
Comment (mickaelandrieu) [inline composer.json]: @eric-chau I'm not sure I need it here, this is the first composer script I did and I don't know well.

File: composer.json [hunk 2]
```diff
@@ -13,13 +13,36 @@
...
+        "backbee/BbCoreJS": "master",
+        "symfony/filesystem": "~2.3"
     },
     "autoload": {
         "psr-4": {
             "BackBee\\Bundle\\ToolbarBundle\\": ""
         }
     },
+    "scripts": {
+        "post-update-cmd": [
```
Comment (mickaelandrieu) [inline composer.json]: I prefer post-update-cmd but imo I need to execute this code on post-install-cmd too.

File: composer.json (hunk 1 above)
Comment (eric-chau) [inline composer.json]: You did well cause ``ToolbarBundle`` has a real dependency to ``incenteev/composer-parameter-handler``.

File: composer.json (hunk 2 above)
Comment (eric-chau) [inline composer.json]: Yes you must add the ``post install event``.

**Comment by eric-chau** [comment]
:+1: good job, thanks =)

**Pull request merged**